style.css

```diff
@@ -29,20 +30,39 @@
 }
 
 .nav-menu {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  gap: 50px;
+  position: fixed;
+  left: -100%;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+  transition: 0.1s;
+  background: rgba(96, 112, 255, 0.7);
+  backdrop-filter: blur(0.5rem);
+  padding-top: 50px;
+  z-index: 9;
+}
+
+.nav-link {
+  font-weight: 600;
+  font-size: 30px;
+  color: #fff;
+  font-family: "Poppins", sans-serif;
+  margin: 10px;
 }
 
 .nav-item {
-  display: none;
+  display: flex;
+  flex-direction: column;
+}
+
+.nav-menu.active {
+  left: 0;
 }
 
 .hamburger {
   display: block;
-  cursor: pointer;
-  margin-right: 20px;
+  padding-right: 20px;
+  z-index: 9999;
 }
 
```

Comment:
- [ **OPTIONAL** ] What do you think if you add the `cursor: pointer` property both on the **Hamburger** and **Close Menu** buttons to help users immediately understand that they are clickable.

Reply:
@tresorsawasawa Thank you for that insightful suggestion.